tests/vmi_cloudinit_test.go

```diff
@@ -380,8 +387,11 @@ var _ = Describe("[rfe_id:151][crit:high][vendor:cnv-qe@redhat.com][level:compon
 
 			})
 			It("[test_id:3183]should have cloud-init network-config from k8s secret", func() {
-				vmi := tests.NewRandomVMIWithEphemeralDiskAndUserdataNetworkData(
-					cd.ContainerDiskFor(cd.ContainerDiskCirros), "", "", false)
+				vmi := libvmi.NewCirros(
+					libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding()),
+					libvmi.WithNetwork(v1.DefaultPodNetwork()),
+					libvmi.WithCloudInitNoCloudUserData(""),
```

Comment:
As you mention in the commit message, NewCirros sets cloudinit once; here it is erased; and third time sets it to a `NetworkDataSecretRef`. It can be nicer with a helper function `withCloudInitNoCloudNetworkDataSecret(secretName)`